mtags-shared/src/main/scala/scala/meta/internal/semver/SemVer.scala

```diff
@@ -38,6 +38,11 @@ object SemVer {
 
     def >=(that: Version): Boolean = this > that || this == that
 
+    def compare(that: Version): Int =
+      if (this > that) 1
+      else if (this < that) -1
+      else 0
+
```

Comment:
I think if you extend smth like `Ordered` you'd have to only implement `compare` to have all the `<` and so on... a bit less boilerplate.